package.json

```diff
@@ -172,8 +172,8 @@
     "supports es6-module and supports css-variables and last 2 versions"
   ],
   "optionalDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
I'm not sure what this actually means, but I think we should support at least react 17+, maybe even more. Is this still possible with this change?

Reply:
Yes, I tested this using liquid-sandbox-react-tailwind linking liquid via npm. No issues.

Reply:
Feel free to merge as soon as you updated the dependencies as talked about in the daily.